Unable to build

I was unable to build the project using `cargo build` from master branch:

```
$ cargo build
...
   Compiling mdbook-d2 v0.2.1 (/home/dompegam/ws/mdbook-d2)
error[E0277]: the trait bound `Value: From<toml::map::Map<std::string::String, toml::value::Value>>` is not satisfied
  --> src/backend.rs:69:34
   |
69 |         let value: toml::Value = ctx.config.get_preprocessor("d2").unwrap().clone().into();
   |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ---- required by a bound introduced by this call
   |                                  |
   |                                  the trait `From<toml::map::Map<std::string::String, toml::value::Value>>` is not implemented for `Value`
   |
   = help: the following other types implement trait `From<T>`:
             <Value as From<&'a str>>
             <Value as From<BTreeMap<S, V>>>
             <Value as From<HashMap<S, V>>>
             <Value as From<Vec<V>>>
             <Value as From<bool>>
             <Value as From<f32>>
             <Value as From<f64>>
             <Value as From<i32>>
           and 7 others
   = note: required for `toml::map::Map<std::string::String, toml::value::Value>` to implement `Into<Value>`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `mdbook-d2` due to previous error
```

Based on some googling it seems that it seems from a change in the toml crate. I downgrade it to `0.5.10` in `Cargo.toml` and was able to build fine. However my rust-foo is not good enough yet to figure out a way to solve it.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,11 +14,11 @@ clap = { version = "4.1", features = ["derive"] }
 serde_json = "1.0"
 semver = "1.0"
 
-mdbook = "0.4"
+mdbook = "0.4.28"
 pulldown-cmark = "0.9"
 pulldown-cmark-to-cmark = "10.0"
 serde = { version = "1.0.152", features = ["derive"] }
-toml = "0.7"
+toml = "0.5"
 anyhow = "1.0.69"
 
 [dev-dependencies]

